api/v1/views/index.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+"""This module defines the index"""
+
+from . import app_views
+from flask import jsonify
+from models.amenity import Amenity
+from models.state import State
+from models.place import Place
+from models.city import City
+from models.review import Review
+from models.user import User
+from models import storage
+
+CLASSES = {
+    'amenities': Amenity,
+    'states': State,
+    'places': Place,
+    'cities': City,
+    'reviews': Review,
+    'users': User
+}
+
+
+@app_views.route('/status')
+def status():
+    """
+    Method that returns a Json with an Ok status
+    """
+
+    return jsonify({'Status': 'OK'})
+
+
+@app_views.route('/stats')
+def stats():
+    """
```

Comment:
Here is missing the documentation

Reply:
Ok! I will add the required documentation

Reply:
You are 🐱‍🏍 great, thanks you the speed you correct this